src/hotspot/share/opto/loopnode.cpp

```diff
@@ -844,9 +844,18 @@ bool PhaseIdealLoop::create_loop_nest(IdealLoopTree* loop, Node_List &old_new) {
   }
 
   // May not have gone thru igvn yet so don't use _igvn.type(phi) (PhaseIdealLoop::is_counted_loop() sets the iv phi's type)
```

Comment:
Comment seems to be outdated as we are not querying the type of the phi anymore.

Reply:
Right. I updated it.